api/services/ZoomAccount/index.js

```diff
@@ -10,22 +11,35 @@ export default class ZoomAccount extends BaseModelService {
     return Models.ZoomAccount.findAll(utilizedResourceClause(...args));
   }
 
-  static async findAvailaibilityDuring(id, ...args) {
-    const data = await Models.ZoomAccount.findOne({
-      where: { id },
-      ...utilizedResourceClause(...args),
-    });
+  static async findAllAvailabilityDuring(keys) {
+    const { id, ...args } = keys[0];
+    return Models.ZoomAccount.findAll({
+      where: {
+        id: {
```

Comment:
```javascript
where {
  [Models.Sequelize.Op.or]: keys.map( (id, dateTimeRange, ...options) => ({
    id,
    ...utilizedResourceClause(dateTimeRange, options)
  })),
}
```

Reply:
You probably mean this right (destruct key)
```
where {
  [Models.Sequelize.Op.or]: keys.map( ({id, dateTimeRange, ...options}) => ({
    id,
    ...utilizedResourceClause(dateTimeRange, options)
  })),
}
```

But this is also isn't working. Probably because `include` do not come in `where` ?

Reply:
`dateTimeRange` and `options` will be same for all the keys right?

We did something similar to this here too **[calender-event-invite-status](https://github.com/coding-blocks/gondor/blob/3775f05141b6cf5041f36cd00e0df3ddfc7cf867/api/services/CalendarEventInvite/index.js#L64)**

Reply:
You don't know if the dateTimeRange will be same or not. There we named it ViewerCalendarEventInviteStatus to reflect that it is for the viewer thus the user_id is same for all.

Reply:
My bad, yeah they will be 2 differ things. but have similar Or logic for the where clause on the first model and the where clause inside the include.